src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -51,16 +64,56 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public Map<String, Long> getCountByRoadSurfaceCondition7(){
-        return null;
+        Map<String, Long> result = new HashMap<>();
+        Multiset<String> roadSurfaceCondition = HashMultiset.create();
```

Comment:
Cool - nice usage of multi collections. I was thinking more about counter as a map value, but this approach is much better than mine :)